packages/expo/src/proxies/NativeModules.ts

```diff
@@ -0,0 +1,90 @@
+import { ExecutionEnvironment } from 'expo-constants';
+
+const PROPS_TO_IGNORE: Set<string> = new Set([
+  /**
+   * We don't want to throw when the expo or expo-modules-core packages try to access any of these
+   * modules, since they have built-in fallbacks.
+   */
+  'DevLoadingView',
+  'EXDevLauncher',
+  'EXReactNativeEventEmitter',
+  'NativeUnimoduleProxy',
+  /**
+   * Other methods that can be called on the NativeModules object that we should ignore. The
+   * underlying NativeModules object is sometimes a proxy itself so may not have these methods
+   * defined.
+   */
+  'hasOwnProperty',
+  'isPrototypeOf',
```

Comment:
Another method to add:
```suggestion
  'isPrototypeOf',
  'propertyIsEnumerable',
```

Reply:
Good catch, thanks